redbeat/decoder.py

```diff
@@ -40,7 +40,7 @@ def dict_to_object(self, d):
             # Decode timestamp values into datetime objects
             for key in ['dtstart', 'until']:
                 if d[key] is not None:
-                    d[key] = datetime.fromtimestamp(d[key])
+                    d[key] = datetime.utcfromtimestamp(d[key])
```

Comment:
Should we be reusing [from_timestamp() and to_timestamp()](https://github.com/sibson/redbeat/blob/72e13c91ce267f6cb86acd71bda2b024e335288e/redbeat/schedulers.py#L73-L81)

Reply:
We should. I will move them to decoder.py to avoid circular dependency.